web/page/javascript.js

```diff
@@ -0,0 +1,30 @@
+const form = document.querySelector('.form')
```

Comment:
Consider using an id for your form instead of the form class 😉 
```suggestion
const form = document.getElementById('form')
```